src/com/game/view/TextColor.java

```diff
@@ -0,0 +1,19 @@
+package com.game.view;
```

Comment:
Does this look like it's part of AB#422 in this ticket? This should be on a branch for that USer Story.

Reply:
Fixed